PloneFormGen improvements

maethu opened the pull request:
Several display issues has been resolved...

bierik commented on plonetheme/blueberry/scss/ploneformgen.scss:
```diff
@@ -0,0 +1,47 @@
+      display: block;
+      visibility: hidden;
+
+      @extend .visualClear;
```
Pls use `@include clearfix()`

maethu replied:
@bierik don

bierik commented:
Could you place some screenies?

maethu commented:
Fixed several issues on the add widget form.
<img width="895" alt="screen shot 2016-05-12 at 09 59 04" src="https://cloud.githubusercontent.com/assets/437933/15312376/dff97fd6-1c03-11e6-82f4-09995850cd6f.png">

Fixed several display issues on the rendered/quickedit form like:
- Show label + required sign on one line
- Show input element below the label and not on the same line
- style fieldset differently for formgen, since there was no visual sign for the fieldset.
- Fix buttons margin. 

<img width="945" alt="screen shot 2016-05-12 at 10 31 04" src="https://cloud.githubusercontent.com/assets/437933/15312438/60f223cc-1c04-11e6-84b5-f4117fadc95c.png">

bierik commented:
👍  thank you
